docs/03-04-function-configuration-file.md

```diff
@@ -0,0 +1,125 @@
+          accessStrategies:
+              handler: jwt
+            - config:
+                jwksUrls:
+                    - http://dex-service.kyma-system.svc.cluster.local:5556/keys
+                    - http://dex-service.kyma-system.svc.cluster.local:5556
+                trustedIssuers:
+                    - https://dex.34.90.136.181.xip.io
+                    - https://dex.34.90.136.181.xip.io
+env:
+    - name: REDIS_PASS
+      value: YgJUg8z6eA
+    - name: REDIS_PORT
+      value: "6379"
+    - name: REDIS_HOST
+      value: hb-redis-enterp-6541066a-edbc-422f-8bef-fafca0befea8-redis.testme.svc.cluster.local
+```
+
+## Parameters
+
+See all parameter descriptions:
+
+| Parameter         | Required | Related custom resource | Description                                   |
+| ---------------------------------------- | :------------: | ---------| ---------|
+| **name**              | Yes | Function | Specifies the name of your Function.         |
+| **namespace**             | No | Function | Defines the Namespace in which the Function is created. It is set to `default` unless you specify otherwise.         |
+| **runtime**             | No | Function | Specifies the execution environment for your Function. The available values are `nodejs12`, `python38` and deprecated `nodejs10`. It is set to `nodejs12` unless specified otherwise.         |
+| **source**            | Yes | Function | Provides details on the type and location of your Function's source code.         |
+| **source.sourceType**            | Yes | Function | Defines that you use inline code or Git repository as the source of the Function's code and dependencies. It must be set either to `inline` or `git`.         |
+| **source.sourcePath**             | Yes | Function | Specifies the absolute path to the directory with the Function's source code.         |
+| **subscriptions.name**           |  Yes | Subscription | Specifies the name of the Subscription custom resource. It takes the name from the Function unless you specify otherwise.    |
+| **subscriptions.protocol**           | Yes  | Subscription | Defines the rules and formats applied for exchanging messages between the components of a given messaging system. Subscriptions in Kyma CLI use the [NATS](https://docs.nats.io/) messaging protocol by default. Must be set to `""`.         |
+| **subscriptions.filter.dialect**            | No | Subscription | Indicates the filter expression language supported by an event producer. Subscriptions specifying the **filter** property must specify the dialect as well. All other properties are dependent on the dialect being used. In the current implementation, this field is treated as a constant which is blank.    |
+| **subscriptions.filter.filters.eventSource**            | Yes | Subscription | The origin from which events are published.         |
+| **subscriptions.filter.filters.eventSource.property**            | Yes | Subscription | Must be set to `source`.         |
```

Comment:
If we are documenting objects like the `subscriptions.filter.filters.eventSource.property` or `source` then maybe we can document tables like the `subscriptions.filter.filters`, `subscriptions` or `apiRules`?

Reply:
I will add thise as well :)